blog/2019-05-28-hola.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
We can remove everything related to blogs, right?

Reply:
i would like to use it as a releaselog with a short paragraph explaining what has been released and any breaking changes

there are a bunch of other bee-based technical writings eg. monthly dev update, devops comms and so on that could  also be included here depending on the functionality available, certainly seems a shame to waste it but let's keep it simple for initial release